### ISSUE TITLE
Purescript not respecting special pursuit syntax

### ISSUE THREAD

**Issue by karljs** [stkb/Rewrap#67]
Most Purescript packages rely on Pursuit to generate/read documentation and the comment syntax for that is `-- | comment here` rather than just `-- comment`.

When I rewrap these kinds of comments, the pipe character is not respected as a comment character:

```
-- | Long comment split
-- | across lines.
```

becomes something like

```
-- | Long comment split | across
-- lines
```

rather than the preferred

```
-- | Long comment split across
-- | lines
```

**Comment by stkb** [comment]
Nice to see a Purescript user :)

Turns out I'd coded for `--|` but not `-- |` with spaces. Have fixed and published v1.6.1. Let me know if there are still problems.

**Comment by karljs** [comment]
@stkb Seems to work great, thanks!
